lib/local-storage.ts

```diff
@@ -104,11 +103,17 @@ if (typeof window !== 'undefined') {
 	}
 } else {
 	// Fallback to filesystem based storage if not in the browser.
-	// tslint:disable-next-line no-var-requires
-	const { LocalStorage } = require('node-localstorage');
```

Comment:
I guess that we are dropping b/c it's on coffeescript and we want to benefit from native es6 classes and/or use the async variants of the fs methods?

Reply:
Being able to use types is a benefit, but primarily using asynchronous fs access is much better. It's not possible for that module to use asynchronous access because localstorage which it's copying is synchronous, but for our needs here we can use either sync or async so we can go with the preferable async access :)